extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -0,0 +1,42 @@
+function renderVideoAccordian(document, videoObjects) {
+    let accordianDiv = document.createElement("DIV");
+    accordianDiv.id = "accordion";
+    videoObjects["video_info"].forEach(video => {
+        let card = document.createElement("DIV");
+        card.className = "card";
+        let cardHeader = document.createElement("DIV");
+        cardHeader.className = "card-header";
+        let videoButton = document.createElement("BUTTON");
+        videoButton.innerHTML = `${video.video_title}`;
```

Comment:
nit: I think `innerText` is better here. Up to you, `HTML` seems to be an overkill for just setting the title